ajax/libs/restful.js/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "restful.js",
+  "filename": "restful.min.js",
+  "version": "0.9.6",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/marmelab/restful.js"
+  },
+  "description": "A pure JS client for interacting with server-side RESTful resources. Think Restangular without Angular.",
+  "author": "Robin Bressan <robin@buddey.net>",
+  "license": "MIT",
+  "dependencies": {
+    "immutable": "3.7.6",
+    "object-assign": "2.0.0",
+    "qs": "5.2.0",
+    "warning": "2.1.0"
+  },
+  "homepage": "http://marmelab.com/blog/2015/03/10/deal-easily-with-your-rest-api-using-restful-js.html",
+  "optionnalDependencies": {
+    "request": "2.69.0",
+    "whatwg-fetch": "0.11.0"
+  },
+  "keywords": [
+    "JavaScript client",
+    "RESTful"
+  ],
```

Comment:
Any reason to drop official keywords?
https://github.com/marmelab/restful.js/blob/master/bower.json#L10-L14